playground/src/Editor/Header.tsx

```diff
@@ -29,77 +25,58 @@ export default function Header({
-              : "text-gray-700 hover:text-gray-900 focus:text-gray-900 dark:text-gray-300 dark:hover:text-white",
-          )}
-          onClick={() => onChangeTab("Source")}
+      <div className="py-4 pl-2">
+        <svg
+          width="136"
+          height="32"
+          viewBox="0 0 272 64"
+          className="fill-galaxy dark:fill-radiate"
+          xmlns="http://www.w3.org/2000/svg"
         >
-          <span
-            className={classNames(
-              "absolute bottom-0 inset-x-0 bg-ayu-accent h-0.5 rounded-full transition-opacity duration-150",
-              tab === "Source" ? "opacity-100" : "opacity-0",
-            )}
+          <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M61.5 0C62.8807 0 64 1.11929 64 2.5V32.06C64 33.4407 62.8807 34.56 61.5 34.56H51.2V39.68H64V64H34.56V44.8H29.44V64H0V0H61.5ZM39.68 29.44V24.32H24.32V29.44H39.68ZM69.12 0H98.56V41.6H103.68V0H133.12V61.5C133.12 62.8807 132.001 64 130.62 64H71.62C70.2393 64 69.12 62.8807 69.12 61.5V0ZM202.24 0H145.86C144.479 0 143.36 1.11929 143.36 2.5V29.44H138.24V53.76H143.36V64H172.8V53.76H199.74C201.121 53.76 202.24 52.6407 202.24 51.26V29.44H172.8V24.32H202.24V0ZM214.98 0H271.36V24.32H241.92V29.44H271.36V51.26C271.36 52.6407 270.241 53.76 268.86 53.76H241.92V64H212.48V53.76H207.36V29.44H212.48V2.5C212.48 1.11929 213.599 0 214.98 0Z"
           />
-          Source
-        </button>
-        <button
-          type="button"
-          className={classNames(
-            "relative flex py-3 text-sm leading-6 font-semibold focus:outline-none",
-            tab === "Settings"
-              ? "text-ayu-accent"
-              : "text-gray-700 hover:text-gray-900 focus:text-gray-900 dark:text-gray-300 dark:hover:text-white",
-          )}
-          onClick={() => onChangeTab("Settings")}
-        >
-          <span
-            className={classNames(
-              "absolute bottom-0 inset-x-0 bg-ayu-accent h-0.5 rounded-full transition-opacity duration-150",
-              tab === "Settings" ? "opacity-100" : "opacity-0",
-            )}
-          />
-          Settings
-        </button>
+        </svg>
       </div>
-      <div className={"flex items-center min-w-0"}>
+      <div className="flex items-center min-w-0">
         {version ? (
-          <div className={"hidden sm:flex items-center"}>
+          <div className="hidden sm:flex items-center">
             <VersionTag>v{version}</VersionTag>
           </div>
         ) : null}
-        <div className="hidden sm:block mx-6 lg:mx-4 w-px h-6 bg-gray-200 dark:bg-gray-700" />
+        <Divider />
         <RepoButton />
-        <div className="hidden sm:block mx-6 lg:mx-4 w-px h-6 bg-gray-200 dark:bg-gray-700" />
+        <Divider />
         <div className="hidden sm:block">
           <ShareButton key={edit} onShare={onShare} />
         </div>
-        <div className="hidden sm:block mx-6 lg:mx-4 w-px h-6 bg-gray-200 dark:bg-gray-700" />
+        <Divider />
```

Comment:
Are the dividers different colors? I assume it's just an artifact of the recording or something.

<img width="384" alt="Screen Shot 2023-07-18 at 8 49 18 PM" src="https://github.com/astral-sh/ruff/assets/1309177/8c4ead11-9153-4dd3-ba4d-6f4f8ee94266">

Reply:
I don't think so... They all render the same HTML 
![image](https://github.com/astral-sh/ruff/assets/1203881/17c33edd-5ba9-4427-ba35-da838a3040be)